saved-sites/saved-sites-store/src/main/java/com/duckduckgo/savedsites/store/SavedSitesEntitiesDao.kt

```diff
@@ -99,7 +99,7 @@ interface SavedSitesEntitiesDao {
     fun entityByUrl(url: String): Entity?
 
     @Query(
-        "select count(*) from entities where entities.url and entities.deleted = 0 LIKE :domain",
+        "select count(*) from entities where entities.url LIKE :domain and entities.deleted = 0 ",
```

Comment:
did that query still work as intended before? 😅

Reply:
...it wasn't crashing 😆